src/System.Private.CoreLib/src/System/Runtime/InteropServices/ComActivator.cs

```diff
@@ -177,6 +186,37 @@ private static Type FindClassType(Guid clsid, string assemblyName, string typeNa
             throw new COMException(string.Empty, CLASS_E_CLASSNOTAVAILABLE);
         }
 
+        private static AssemblyLoadContext GetALC(string assemblyName)
+        {
+            if (s_AssemblyLoadContexts == null)
```

Comment:
Technically s_AssemblyLoadContexts should be read with a volatile, though in practice on coreclr I don't think it'll matter.  You could address that and consolidate with the below with:
```C#
return LazyInitializer.EnsureInitialized(ref s_AssemblyLoadContext).TryGetValue(assemblyName, out AssemblyLoadContext alc)) ?
    alc :
    AddNewValue(assemblyName, new IndividualAssemblyLoadContext());
```